make error v7.42 (master) RuntimeError: Digest mismatch

XCSoar versions having and not having the problem
-------------------------------------------------

v7.42

System information
------------------

build Unix or KOBO version

Steps to reproduce the behavior
-------------------------------

- make clean
- make -j5 TARGET=KOBO DEBUG=n output/KOBO/KoboRoot.tgz

Expected behavior
-----------------

should compile without error

Actual behavior
---------------

raise RuntimeError("Digest mismatch")
RuntimeError: Digest mismatch
make: *** [build/libboost.mk:14: output/download/boost_1_84_0.tar.bz2] Fehler 1


Do you have any idea what may have caused this?
-----------------------------------------------

in /home/xxxxx/XCSoar/build/libboost.mk we have the following 2 lines

BOOST_URL = https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2
BOOST_ALTERNATIVE_URL = https://sourceforge.net/projects/boost/files/boost/1.84.0/boost_1_84_0.tar.bz2/download


Do you have an idea how to solve the issue?
-------------------------------------------

it is possible to download the tar file in BOOST_ALTERNATIVE_URL
but trying to download the file mentioned in BOOST_URL we get as result a redirection to
https://landing.jfrog.com/reactivate-server/boostorg

No idea what is necessary here

--- a/build/libboost.mk
+++ b/build/libboost.mk
@@ -1,4 +1,4 @@
-BOOST_URL = https://boostorg.jfrog.io/artifactory/main/release/1.84.0/source/boost_1_84_0.tar.bz2
+BOOST_URL = https://fossies.org/linux/misc/boost_1_84_0.tar.bz2
 BOOST_ALTERNATIVE_URL = https://sourceforge.net/projects/boost/files/boost/1.84.0/boost_1_84_0.tar.bz2/download
 BOOST_MD5 = cc4b893acf645c9d4b698e9a0f08ca8846aa5d6c68275c14c3e7949c24109454
 

